conf/repo/mail.yml

```diff
@@ -0,0 +1,8 @@
+mail:
+  branches:
+    - "18.0"
+  category: Mail
+  default_branch: "18.0"
+  name: Addons related to odoo's messaging features
```

Comment:
@simahawk @etobella is this name field supposed to be synced to the repo somewhere? I expected it in About but I see only `mail` over there.

Reply:
I guess we miss something here:
- repo creation https://github.com/OCA/repo-maintainer/blob/master/oca_repo_maintainer/tools/manager.py#L145 
- branch init https://github.com/OCA/repo-maintainer/blob/master/oca_repo_maintainer/tools/manager.py#L191

Reply:
Create #54 to track this.